### DISCUSSION TITLE
Scroll Area

### DISCUSSION THREAD

**Discussion by saeedsv** [quasarframework/quasar#10729] [General - Components / Directives / etc]
Then **scrollbar**  part of  `q-scroll-area`,  for every scroll position; remains at right side of component  if quasar **rtl** mode is true

**Comment by ontwikkelfabriek** [comment]
@saeedsv 

If you think this is a bug , please create an issue or a codepen to demo your problem

**Comment by pdanpdan** [comment]
Only in Quasar v2 or in Quasar v1 also?

**Comment by ontwikkelfabriek** [comment]
@pdanpdan 

I am not having this issue in Qv1  and Qv2(beta1)...

**Comment by pdanpdan** [comment]
In that case please open an issue - it must be a regression. Thank you
